bbs/readmail.cpp

```diff
@@ -322,9 +322,9 @@ static std::string from_name(const mailrec& m, const Network& net, const slrec&
       return a()->names()->UserName(m.fromuser);
     }
   } else {
-    if (nn == 255) {
-      return fmt::format("#{} @{}.", m.fromuser, m.fromsys, net.name);
-    }
+//    if (nn == 255) {
```

Comment:
why?

Reply:
this is purely cosmetic. if we leave this in the output shows "#2 @deleted network #5" in email header listing, but since the email body/text actually has the name of sender etc, letting it pass through to the other header display routines still works.

Reply:
LGTM